pkg/rancher-desktop/main/update/LonghornProvider.ts

```diff
@@ -101,7 +101,7 @@ export type UpgradeResponderRequestPayload = {
   },
 };
 
-export interface GithubReleaseAsset {
+export interface GitHubReleaseAsset {
```

Comment:
If this is a public API, it shouldn't be renamed. If not, it's a brand and should be fixed.

Reply:
It's public only within the code-base, and can be renamed.

Reply:
In fact I don't think it needs to be exported. Asking Mark...